documentation/sphinx/source/backups.rst

```diff
@@ -77,31 +77,40 @@ For local directories, the Backup URL format is
-      <hostname> - Remote hostname or IP address to connect to
-      <port> - Remote port to connect to.  Optional.  Default is 80.
-      <name> - Name of the backup within the backup bucket.  It can contain '/' characters in order to organize backups into a folder-like structure.
-      <bucket_name> - Name of the bucket to use for backup data.
-      <region_name> - If <hostname> is not in s3 compatible form (s3.region-name.example.com) and aws v4 signature is enabled, region name is required.
-      
-      <param>=<value> - Optional URL parameters.  See below for details.
+      <api_key>         API key to use for authentication. If S3, it is AWS_ACCESS_KEY_ID. Optional.
+      <secret>          API key's secret.  If S3, it is AWS_SECRET_ACCESS_KEY. Optional.
+      <security_token>  Security token if temporary credentials are used. If S3, it is AWS_SESSION_TOKEN. Optional.
+      <hostname>        Remote hostname or IP address to connect to
+      <port>            Remote port to connect to.  Optional.  Default is 80.
+      <name>            Name of the backup within the backup bucket.  It can contain '/' characters in order to organize backups into a folder-like structure.
+      <bucket_name>     Name of the bucket to use for backup data.
+      <region_name>     If <hostname> is not in s3 compatible form (s3.region-name.example.com) and aws v4 signature is enabled, region name is required.
+      <param>=<value>   Optional URL parameters.  See below for details.
 
-A single bucket (specified by <bucket_name>) can hold any number of backups, each with a different <name>.
 
-If <secret> is not specified, it will be looked up in :ref:`blob credential sources<blob-credential-files>`.
+The <secret> may be omitted from the blob URL and instead obtained from :ref:`blob credential sources<blob-credential-files>`.
+
+.. _blob-url-examples:
 
 An example blob store Backup URL would be ``blobstore://myKey:mySecret@something.domain.com:80/dec_1_2017_0400?bucket=backups``.
 
+If S3 is the target blobstore, the URL would look like: ``blobstore://AWS_ACCESS_KEY_ID:AWS_SECRET_ACCESS_KEY:AWS_SESSION_TOKEN@backup-12345-us-west-2.s3.amazonaws.com/dailies?bucket=backup-12345-us-west-2&region=us-west-2``.
+Replace the ``AWS_...`` placeholders with their values obtained from your aws context (the URL will be looooonnnng!).
+
+The S3 <secret> (and <security_token>) may be ommitted from the URL and instead picked up from :ref:`blob credential sources<blob-credential-files>` as noted above.
+The URL would then look like: ``blobstore://AWS_ACCESS_KEY_ID@backup-12345-us-west-2.s3.amazonaws.com/dailies?bucket=backup-12345-us-west-2&region=us-west-2``.
+
+To pickup the S3 <api_key>, <secret>, and <security_token> from :ref:`blob credential sources<blob-credential-files>`,
+write the blob backup URL as: ``blobstore://@backup-12345-us-west-2.s3.amazonaws.com/dailies?bucket=backup-12345-us-west-2&region=us-west-2`` (Notice the '@' in front of the hostname!).
```

Comment:
```suggestion
write the blob backup URL as: ``blobstore://@backup-12345-us-west-2.s3.amazonaws.com/dailies?bucket=backup-12345-us-west-2&region=us-west-2`` (Notice the '@' in front of the hostname).
```

Reply:
Thanks for the review.

I dropped the certs and keys from the example and extra  --knob_blobstore_encryption_type=aws:kms knob.

The certs/key not needed as you suggest in this case and ' --knob_blobstore_encryption_type=aws:kms' probably just confuses.

Hopefully I got it right.